kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/dto.scala

```diff
@@ -32,6 +32,11 @@ case class SessionOverview(
   sessionHandle: SessionHandle
 )
 
+case class InfoValue(
```

Comment:
`InfoOverview` or `InfoDetail` sounds better? Because you have a field named `infoValue `, so it is a bit confusing. WDYT?

Reply:
Make sense, updated to InfoDetail.